faculty/clients/job.py

```diff
@@ -125,6 +150,73 @@ def make_job_summary(self, data):
         return JobSummary(**data)
 
 
+class InstanceSizeSchema(BaseSchema):
+    milli_cpus = fields.Float(data_key="milliCpus", required=True)
+    memory_mb = fields.Float(data_key="memoryMb", required=True)
+
+    @post_load
+    def make_instance_size(self, data):
+        return InstanceSize(**data)
+
+
+class JobParameterSchema(BaseSchema):
+    name = fields.String(required=True)
+    param_type = EnumField(
```

Comment:
Why not just `'type`?

If it's because 'type' has special meaning in Python, I think it's ok to use it as an attribute as it's a builtin function, not a keyword, and as an attribute it won't conflict with the `type` builtin.